FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -102,15 +118,15 @@ void printToWeb (double precision){
     log_info( "Calibration Precision: " << precision << "mm");
 }
 
-void Maslow_::readEncoders() {
-  axisTL.readEncoder();
-  axisTR.readEncoder();
-  axisBL.readEncoder();
-  axisBR.readEncoder();
-}
+// void Maslow_::readEncoders() {
```

Comment:
Was this just not being called anywhere?

Reply:
yep